ansible/management-searchengine.yml

```diff
@@ -0,0 +1,298 @@
+# Search ngine + Search Engine Client + Elasticsearch
```

Comment:
Optionally, this playbook could also be renamed as `idr-searchengine.yml` since it's no longer tied to the management VM